biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1118,8 +1041,18 @@ subroutine spawn_patches( currentSite, bc_in)
              call sort_cohorts(currentPatch)
              
              !update area of donor patch
+             oldarea = currentPatch%area
              currentPatch%area = currentPatch%area - patch_site_areadis
 
+             ! for all disturbance rates that haven't been resolved yet, increase their amount so that 
+             ! they are the same amount of gridcell-scale disturbance relative to the original patch size
+             if (i_disturbance_type .ne. N_DIST_TYPES) then
+                do i_dist2 = i_disturbance_type+1,N_DIST_TYPES
+                   currentPatch%disturbance_rates(i_dist2) = currentPatch%disturbance_rates(i_dist2) &
+                        * oldarea / currentPatch%area
+                end do
+             end if
```

Comment:
Following through the logic here with a generic example, and I think it checks out:

lets assume that the patch starts out with an area of 1., and all three types apply 0.333 disturbance rates.

After the first disturbance, we have an area of 0.666, and the remaining two rates are bumped up to 0.333*(1/0.666) = 0.5.

After the second disturbance, we have an area of 0.33 = 0.66 * 0.5, and the remaining disturbance rate is bumped up to 0.5*(0.66/0.33) = 1.0  

The last disturbance will cover the entire remaining area fraction.